build.sbt

```diff
@@ -434,6 +436,16 @@ lazy val `doc-examples` = project
     publish / skip := true,
     Dependencies.`Doc-examples`)
 
+lazy val billOfMaterials = Project("bill-of-materials", file("bill-of-materials"))
+  .enablePlugins(BillOfMaterialsPlugin)
+  .disablePlugins(MimaPlugin, SitePlugin)
+  .settings(
+    name := "pekko-connectors-bom",
+    licenses := List(License.Apache2),
+    libraryDependencies := Seq.empty,
```

Comment:
`libraryDependencies` default value is Seq.empty? if true we could ignore this.

Reply:
this is needed to fight against the commonSettings in the build - that hardcodes a dependency on pekko-stream.
BOMs don't have dependencies.